lib/relationships.js

```diff
@@ -130,3 +161,17 @@ function updateRelation (model, id, data) {
     }
   })
 }
+
+function keyByModel(assocModel, model) {
+  var key = null;
```

Comment:
Is this the right name for this function? 
Am I right in thinking it gets the fk on the through model that points at the to model?